nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    nil
+  end
```

Comment:
- [ ] Kindly make sure that this method returns a `NotImplementedError`,as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Implement%20a%20method%20called%20correct_name%20that%20will%20raise%20a%20NotImplementedError.) requirement.